pkg/gardenlet/controller/shoot/shoot_control_delete.go

```diff
@@ -459,12 +458,12 @@ func (c *Controller) runDeleteShootFlow(o *operation.Operation) *gardencorev1bet
 		)
 
 		destroyInternalDomainDNSRecord = g.Add(flow.Task{
-			Name:         "Destroying internal DNS entry",
-			Fn:           flow.TaskFn(component.OpWaiter(botanist.Shoot.Components.Extensions.DNS.InternalEntry).Destroy),
+			Name:         "Destroying internal domain DNS record",
+			Fn:           flow.TaskFn(botanist.DestroyInternalDNS),
 			Dependencies: flow.NewTaskIDs(syncPoint),
 		})
 		deleteDNSProviders = g.Add(flow.Task{
-			Name:         "Deleting DNS providers",
+			Name:         "Deleting additional DNS providers",
```

Comment:
Is this step seems unnecessary considering the providers are deleted in DestroyInternalDNS and DestroyExternalDNS functions?